fbu-app/PostSearch/ViewControllers/HouseBalancesViewController.m

```diff
@@ -0,0 +1,159 @@
+//
+//  HouseBalancesViewController.m
+//  fbu-app
+//
+//  Created by sophiakaz on 8/3/19.
+//  Copyright © 2019 lucjia. All rights reserved.
+//
+
+#import "HouseBalancesViewController.h"
+#import "HousemateBalanceCell.h"
+#import "House.h"
+#import "Persona.h"
+#import "Parse/Parse.h"
+#import "Balance.h"
+
+@interface HouseBalancesViewController ()<UITableViewDataSource, UITableViewDelegate>
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (weak, nonatomic) IBOutlet UILabel *negativeLabel;
+@property (weak, nonatomic) IBOutlet UILabel *positiveLabel;
+@property (weak, nonatomic) IBOutlet UILabel *totalLabel;
+@property (nonatomic, strong) NSMutableArray *balances;
```

Comment:
the non-iboutlets dont need to be made properties, they can just be instance variables

not sure if IBOutlets dont't have to be made properties